src/components/blocks/blocks.css

```diff
@@ -4,4 +4,5 @@
     right: 0;
     bottom: 0;
     left: 0;
-}
+    border-radius: 0.75rem;
+}
```

Comment:
Missing ending line breaks at EOF in several places (just marking this one). 

Sublime (I think that is what you use?) can be configured to do this automatically:
https://forum.sublimetext.com/t/make-saving-newline-at-eof-the-installation-default/9842